exe/wallet/Main.hs

```diff
@@ -313,9 +318,15 @@ exec execServer manager args
         -> ClientM a
         -> IO ()
     runClient encode cmd = do
+        res <- sendRequest cmd
+        handleResponse encode res
+
+    sendRequest cmd = do
```

Comment:
@KtorZ I did what you suggested, but failed with providing function declaration for both `sendRequest` and `handleResponse`, tried this, but it didn't work:

```haskell
    handleResponse
        :: forall a. ()
        => (a -> BL.ByteString)
        -> IO (Either ServantError a)
        -> IO ()


    sendRequest
        :: forall a. ()
        => ClientM a
        -> IO (Either ServantError a)
```